src/Operation/EstimatedDocumentCount.php

```diff
@@ -157,12 +166,9 @@ private function createAggregate(): Aggregate
         );
     }
 
-    /** @return Aggregate|Count */
-    private function createCommand(Server $server)
+    private function createCommand(Server $server): Count
     {
-        return server_supports_feature($server, self::$wireVersionForCollStats)
-            ? $this->createAggregate()
-            : $this->createCount();
+        return $this->createCount();
```

Comment:
With this change, I believe `createAggregate` is no longer called and should be removed.